### DISCUSSION TITLE
Anybody got this working on Linux?

### DISCUSSION THREAD

**Discussion by felixpelletier** [sultim-t/xash-rt#67] [Q&A]
I tried running this using Proton, but it always throws an error "Host_InitError: Can't initialize any renderer. Check your video drivers!"
There's no error in the logs or anything.

I understand there's no Linux version of the mod, but if someone's got an easy fix for running this in Linux, that'd be awesome. :D

**Comment by zt64** [comment]
To get it running on Linux you need to install vcrun19 using winetricks, or protontricks. After that it works flawlessly from my testing.

**Comment by Nord1cWarr1or** [comment]
I just ran the game on Linux, using PortProton
